modules/mesh-layers/src/scenegraph-layer/scenegraph-layer.js

```diff
@@ -61,36 +64,30 @@ const fs = `
     varying vec2 vTEXCOORD_0;
     uniform sampler2D u_BaseColorSampler;
   #endif
+  varying vec4 vColor;
 
   void main(void) {
     #ifdef HAS_UV
-      gl_FragColor = texture2D(u_BaseColorSampler, vTEXCOORD_0);
+      gl_FragColor = (vColor / 255.) * texture2D(u_BaseColorSampler, vTEXCOORD_0);
     #else
-      gl_FragColor = vec4(0, 0, 0, 1);
+      gl_FragColor = vColor / 255.;
     #endif
 
     gl_FragColor = picking_filterPickingColor(gl_FragColor);
   }
 `;
 
-const DEFAULT_COLOR = [0, 0, 0, 255];
-const RADIAN_PER_DEGREE = Math.PI / 180;
+const DEFAULT_COLOR = [255, 255, 255, 255];
 
-const defaultProps = {
-  sizeScale: {type: 'number', value: 1, min: 0},
+const defaultProps = Object.assign(
+  {
+    sizeScale: {type: 'number', value: 1, min: 0},
 
-  getPosition: {type: 'accessor', value: x => x.position},
```

Comment:
I am not super keen on pushing these into another file. 

I see the reuse value, but I also see a lot of value in being able to scan all props here without chasing into other files.

Reply:
+1. The defaultProps here are also serving as documentation.